CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 ## 10.4.0 (YYYY-MM-DD)
+### Breaking Changes
```

Comment:
So this has to bump the major version?

Reply:
All of these changes are part of the datatype release which will be a major version bump